src/params.ts

```diff
@@ -115,9 +102,6 @@ export const releaseFilesDefaultNames: {
   compose: "docker-compose.yml",
   avatar: "avatar.png",
   setupWizard: "setup-wizard.json",
-  setupSchema: "setup.schema.json",
-  setupTarget: "setup-target.json",
-  setupUiJson: "setup-ui.json",
```

Comment:
We cannot merge this change until there's hard proof that no package is using the old setup wizard v1 feature